src/Common/Tooltip/Tooltip.tsx

```diff
@@ -1,24 +1,26 @@
-import { ITooltipHostStyles, TooltipHost } from "@fluentui/react";
+import { IButtonStyles, IconButton, ITooltipHostStyles, TooltipHost } from "@fluentui/react";
 import { useId } from "@fluentui/react-hooks";
 import * as React from "react";
-import InfoBubble from "../../../images/info-bubble.svg";
 
 const calloutProps = { gapSpace: 0 };
 const hostStyles: Partial<ITooltipHostStyles> = { root: { display: "inline-block" } };
 
 export interface TooltipProps {
   children: string;
 }
+
+const iconButtonStyles: Partial<IButtonStyles> = { root: { marginBottom: -3 } };
+const iconProps = { iconName: "Info" };
+
 export const Tooltip: React.FunctionComponent = ({ children }: TooltipProps) => {
```

Comment:
To be honest, I don't see why we have to define a separate component for this. This is basically the `TooltipHost` component from Fluent UI with some extra styling. For different icon types (e.g. info vs. error icon), the styling should be different anyways so it doesn't really make sense to make this abstraction.

Reply:
done